plugins/github-auth-plugin/src/github-auth-plugin.ts

```diff
@@ -0,0 +1,35 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import * as theia from '@theia/plugin';
+import * as che from '@eclipse-che/plugin';
+
+export function start(context: theia.PluginContext) {
+    if (theia.plugins.getPlugin('github.vscode-pull-request-github')) {
+        const informationMessageTestCommand = {
```

Comment:
looks the variable name is not self explanatory

Reply:
fixed